quiz.js

```diff
@@ -0,0 +1,56 @@
+// THESE ARE THE MAIN ELEMENTS FOR EQUATION
+var height = document.getElementById("tree_height");
+var characterOption = document.getElementById("charct_opt");
+var submitButton = document.getElementById("getTree");
+
+
+// UPON "CLICK" OR "KEYPRESS"(ENTER)- THE SUBMIT BUTTON WILL ENGAGE
+submitButton.addEventListener("click", correctInput);
+addEventListener("keypress", checkReturn);
+
+function checkReturn (e){
+  if (e.keyCode == 13){
+   correctInput()
+  }
+}
+
+// ****TESTING AND ADDING COMMENTS IN CASE SOMEONE DOES NOT INPUT PROPERLY**** PERSONAL NOTE: This came easy!
+
+function correctInput () {
+  if (height.value >= 100) {
+    alert("Please enter value lower than 100. You may choose any character you please for the next input field.");
+} else  {
+    height = height.value;
+    characterOption = characterOption.value;
+    generateTreeObject(height, characterOption);
+        };
+}
+
+
+// TW PERSONAL NOTE: Creating the object and deleted my prior wrong doing of height * charoption (<<<--not correct). Create an object, console.log it, and create the next function.
+// think of it as information traveling from one block to the next!! (TW-Study more on this)
+
+function generateTreeObject (height, characterOption) {
```

Comment:
Can you eliminate this function?  As a rule of thumb, if your function is just calling another function, it's probably extraneous.